### ISSUE TITLE
[deeplab v3+] training details

### ISSUE THREAD

**Issue by tonghe90** [tensorflow/models#4345]
It is said the size for calculating bn param is 16 (12 at least). What about the total batch size during the training process for voc trainaug and coco, separately.

**Comment by aquariusjay** [comment]
Hi tonghe90,

As mentioned in the document, using batch size 16 should be sufficient. Feel free to try even larger batch size and we would be happy to hear the feedback.

**Comment by tt-leader** [comment]
@tonghe90 I'm trying to reproduce deeplab v3+ too, could you please tell me how many gpu memory and how many gpu cards did you use when using parameter batch_size = 16, crop_size=769?

**Comment by tonghe90** [comment]
@tt-leader we use 16g memory(per card) to put 16 images for batch normalization and the total batch size is 32 or 64 if you want to train coco. the crop size is used with default 513.

**Comment by nemonameless** [comment]
@tt-leader   Have you  reproduced deeplab v3+ ?   I only train on 4 gpu(every one is 12g),   batch_size = 8, get 74.00%  ,  how to set my parameters to get  78.79% in [https://github.com/tensorflow/models/blob/master/research/deeplab/g3doc/model_zoo.md] ? thx

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
